aws/validators.go

```diff
@@ -654,6 +655,55 @@ func validateAwsAccountId(v interface{}, k string) (ws []string, errors []error)
 	return
 }
 
```

Comment:
Could you please move these validators into the resource's source file (`resource_aws_kinesis_analyticsv2_application.go`)?